web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1100,6 +1100,35 @@ $.extend(fixmystreet.set_up, {
     update_label('#filter_categories', translation_strings.select_category_aria_label);
   },
 
+  has_selector_checker: function() {
+    var supportsHas = CSS.supports('selector(:has(*))');
+
+    if (!supportsHas) {
+        $('.govuk-multi-select__label').each(function() {
+            var label = $(this);
+            var input = label.find('input[type="checkbox"], input[type="radio"]');
+      
+            if (input.attr('type') === 'checkbox') {
+              label.addClass('govuk-multi-select__label--checkbox');
+            } else if (input.attr('type') === 'radio') {
+              label.addClass('govuk-multi-select__label--radio');
+            }
+      
+            if (input.prop('checked')) {
+              label.addClass('govuk-multi-select__label--checked');
+            }
+      
+            input.on('change', function() {
+              if (this.checked) {
```

Comment:
This is fine, but you can also have this as one line with:
`label.toggleClass('govuk-multi-select__label--checked', this.checked);`